aws-ts-apigateway/README.md

```diff
@@ -1,6 +1,6 @@
 [![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
 
-# Serverless REST API on AWS
+# Serverless REST API
```

Comment:
Some titles keep "AWS" in the title. Not pushing back, just curious why we're removing it here and in other places "on AWS" is used.

Reply:
I was torn on some titles that are really short like our basic examples for provisioning resources. It made sense to leave the cloud provider in, but the directory structure (in the repo) and the breadcrumbs will probably make this redundant. Might be best to just drop "AWS", "Azure", and "GCP" from the titles.

Reply:
Could the rule be "it's okay to add names of specific services (S3, EC2), but don't use blank 'AWS'"?

Reply:
Yes, that is correct. I'll clarify that in the example template.